app/_hub/kong-inc/file-log/index.md

```diff
@@ -78,134 +79,14 @@ params:
 
 ---
 
-## Log Format
+## Log format
 
-Every request will be logged separately in a JSON object separated by a new line `\n`, with the following format:
+{% include /md/plugins-hub/log-format.md %}
```

Comment:
I applaud the decision of avoiding repetition by using includes like this one, it will help us maintain the plugin pages in the future.

However, I think we should put some sort of version number on this shared file.

The reason is that the log format (slowly) evolves with time. It's in fact one of the changes we'll introduce in 2.4 - new options for customizing the log format.

While it will not change from version to version, I suggest renaming the include file so this include becomes:

```suggestion
{% include /md/plugins-hub/log-format-kong-2.3.md %}
```

In future versions we can create a `log-format-kong-2.4.md` but still retain the old formats for older versions of the plugin. We also leave the door open for EE to have a slightly different log format in EE log plugins if they need to.

Reply:
@kikito the problem that I think will emerge with automation is that the plugin hub doc source is not currently in a versioned area of the repo. Please see my comment to Murillo's in the plugin spreadsheet https://docs.google.com/spreadsheets/d/1VZY6KZZDWhqYqq0s9arZyrNLdzcUpwCDaOVj9oS_DII/edit?disco=AAAALomCZd4.  I have mentioned to the docs team and product management that we need properly versioned repos, not everything including the kitchen sink in a master repo only. And a separate repo for docs that aren't versioned in the traditional release sense, such as Konnect kloud things. I do have a comment as to which version the log format is from in the source file. 

![image](https://user-images.githubusercontent.com/3756245/110492559-eb4d1500-80b7-11eb-96c8-daa82224f4ac.png)

I think trying to version files in an unversioned area of the docs repo is going to be unwieldy.

cc: @eskibars